cha-ching-app/src/App.js

```diff
@@ -18,18 +18,34 @@ function App() {
       });
   }, []);
 
+  useEffect(() => {
+    const requestOptions = {
+      method: "POST",
```

Comment:
great to know that we can use GET, POST, or PUT with a fetch() through this requestOptions object!